src/components/Popups/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import classnames from 'classnames';
+
+export type Props = {
+    className?: string;
+    defaultValue?: string;
+    onChange: (event: React.ChangeEvent<HTMLTextAreaElement>) => void;
+    onClick: (event: React.SyntheticEvent) => void;
+    textareaRef?: React.Ref<HTMLTextAreaElement>;
+};
+
+class TextArea extends React.Component<Props> {
+    render(): JSX.Element {
+        const { className, textareaRef, ...rest } = this.props;
+
+        return <textarea ref={textareaRef} className={classnames('ba-TextArea', className)} {...rest} />;
+    }
+}
+
+export default TextArea;
```

Comment:
This component will eventually expand to include more functionality. It may not end up using a textarea element under the hood. Maybe a more generic name would work, like `ReplyField`?